ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -123,6 +137,10 @@ export class PrepareMigrationComponent implements OnInit {
           name: 'Minimal downtime Migration',
           value: MigrationTypes.lowDowntimeMigration,
         },
+        {
```

Comment:
Not sure if this is the right places to add dataproc because this.isStreamingSupported is true for oracle, mysql and postgresql and this PR adds dataproc integration only for MySQL

Reply:
Added check to show dataproc migration mode only for MySQL
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/519/commits/250c1607ededfd68eee6f3cf324b820a82699f33